spl/token/client.py

```diff
@@ -283,7 +284,40 @@ def create_wrapped_native_account(
         :param amount: The amount of lamports to wrap.
         :return: The new token account.
         """
-        raise NotImplementedError("create_wrapped_native_account not implemented")
+        new_account = Account()
+        # Allocate memory for the account
+        balance_needed = Token.get_min_balance_rent_for_exempt_for_account(conn)
+        # Construct transaction
+        txn = Transaction()
+        txn.add(
+            sp.create_account(
+                sp.CreateAccountParams(
+                    from_pubkey=payer.public_key(),
+                    new_account_pubkey=new_account.public_key(),
+                    lamports=balance_needed,
+                    space=ACCOUNT_LAYOUT.sizeof(),
+                    program_id=program_id,
+                )
+            )
+        )
+
+        txn.add(
+            sp.transfer(
+                sp.TransferParams(from_pubkey=payer.public_key(), to_pubkey=new_account.public_key(), lamports=amount)
+            )
+        )
+
+        txn.add(
+            spl_token.initialize_account(
+                spl_token.InitializeAccountParams(
+                    account=new_account.public_key(), mint=WRAPPED_SOL_MINT, owner=owner, program_id=program_id
+                )
+            )
+        )
+
+        conn.send_transaction(txn, payer, new_account, opts=TxOpts(skip_preflight=True, skip_confirmation=False))
```

Comment:
Let's not hard code `skip_confirmation` and make this an optional parameter on the function.